certified-connectors/DomainToolsIrisEnrich/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1145 @@
+              },
+              "absolute_limit": {
+                "type": "string",
+                "description": "Absolute Limit",
+                "title": "Absolute Limit"
+              },
+              "usage": {
+                "type": "object",
+                "properties": {
+                  "today": {
+                    "type": "string",
+                    "description": "Today Usage",
+                    "title": "Today Usage"
+                  },
+                  "month": {
+                    "type": "string",
+                    "description": "Month Usage",
+                    "title": "Month Usage"
+                  }
+                },
+                "description": "Usage Details"
+              },
+              "expiration_date": {
+                "type": "string",
+                "description": "Expiration Date",
+                "title": "Expiration Date"
+              }
+            }
+          },
+          "description": "Products Information"
+        }
+      },
+      "description": "Acoount Information Response"
+    },
+    "Enrich_Response": {
+      "type": "object",
+      "properties": {
+        "response": {
+          "type": "object",
+          "properties": {
+            "limit_exceeded": {
+              "type": "boolean",
+              "description": "DomainTools API results are limited to 5000 entries in a result set. If a query has more than 5000 results, the 'Limit exceeded' returns true. Try refining your query with more specific search terms.",
+              "title": "Limit Exceeded"
+            },
+            "message": {
+              "type": "string",
+              "description": "message",
+              "title": "Message"
+            },
+            "results_count": {
+              "type": "integer",
+              "format": "int32",
+              "description": "results_count",
+              "title": "Results Count"
+            },
+            "results": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "domain": {
+                    "type": "string",
+                    "description": "domain",
+                    "title": "Domain"
+                  },
+                  "whois_url": {
+                    "type": "string",
+                    "description": "whois_url",
+                    "title": "Whois URL"
+                  },
+                  "adsense": {
+                    "type": "object",
+                    "properties": {
+                      "value": {
+                        "type": "string",
+                        "description": "Adsense Value",
+                        "title": "Adsense Value"
+                      }
+                    },
+                    "description": "adsense"
+                  },
+                  "alexa": {
+                    "additionalProperties": {},

```

Comment:
@RamboV Empty property values can be removed until unless you have a specific reason to keep them.

Reply:
Hi @sriyen-msft , yes this is left intentionally for a few response properties, as for few of the response properties datatypes are dynamic in nature.